src/azure-cli-core/HISTORY.rst

```diff
@@ -2,6 +2,8 @@
 
 Release History
 ===============
+2.0.59
```

Comment:
Is this simply for the Edge Build?

Reply:
Yep, edge build plus CI is checking for it and dinging me right now.

Reply:
But we are releasing just 2.0.58?

Reply:
Correct. These changes will not be pulled into the release or master branch until code-complete for 2.0.59

Reply:
Cool. Approved. But out of curiosity, what did the CI complain about when the version was 2.0.58?  The CI isn't supposed to do version checks anymore.

Reply:
Ah, sorry the changes that got dinged by CI were changing the version number (for the edge build only) without updating the appropriate `HISTORY.rst`s. (This still surprised me a little, because this doesn't seem to be enforced with the command_modules).